src/client/containers/dashboard-page/dashboard-page.component.js

```diff
@@ -22,6 +22,7 @@ const DashboardPageContainer = () => {
   const [lineChartDataEc, setLineChartDataEc] = useState(null);
```

Comment:
Small thing that should probably have been caight in a previous PR, but it would be a bit more intuitive if the file was called dahboard-page.container.js. If you fix it, make sure to catch any references and update them.